File: MODUL 6/C/PRAK601-2310817220011-ReginaSilvaMaharatini.c
#include<stdio.h>
int main(){

    int baris,kolom ,A ,B ,bil;
  scanf("%d %d", &baris, &kolom);
    int matriks[baris][kolom];
    for(A = 0;  A < baris; A++)
    {
    for(B = 0; B < kolom; B++){
  scanf("%d", &bil);
    matriks[A][B] = bil;}}
    for(A = 0; A < baris; A++)
    {
    for(B = 0; B < kolom; B++){
  printf("%d ", matriks[A][B]);} 
  printf("\n");}
} 

File: MODUL 6/Python/PRAK601-2310817220011-ReginaSilvaMaharatini.py
baris, kolom = map(int, input().split())
angka=[]
num = 0
isi = list(map(int, input().split()))

for i in range (baris):
    angka.append(isi[num:num+kolom])
    num = num + kolom
for baris in angka:
    for isi in baris:
        print(isi, end=' ')
    print()